packages/gestalt-core/build.js

```diff
@@ -65,6 +65,7 @@ const statsPlugin = () => {
 const cssModules = (options = {}) => {
   const cssExportMap = {};
   const scopeNames = {};
+  let cssIE11 = ''; // CSS targeted toward IE11
```

Comment:
I thought about calling this "legacy" but I was only planning on testing in IE11 since that's the main browser we're generating this for. There could be some older Android devices we need to consider on mobile as well https://caniuse.com/#feat=css-variables